services/121-service/src/registration/repositories/registration-scoped.repository.ts

```diff
@@ -177,7 +177,7 @@ export class RegistrationScopedRepository extends RegistrationScopedBaseReposito
   }): Promise<GetDuplicatesResult[]> {
     // Added a limit to the number of duplicates to prevent a large amount of duplicates from being returned which could cause performance issues
     // If there are more than this amount of duplicates a user misconfiguration is likely
-    const maxDuplicates = 10;
+    const maxDuplicates = 9;
```

Comment:
```suggestion
    const maxDuplicates = 10;
```

This feels unintentional and a result of an old rebase. Am I right? If not, I wonder why this change is needed.

Reply:
this should stay at 9 because of backend constraints